src/sentry/migrations/0516_switch_pagerduty_silo.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.20 on 2023-07-18 16:07
+
+import django.db.models.deletion
+from django.db import migrations
+
+import sentry.db.models.fields.foreignkey
+import sentry.db.models.fields.hybrid_cloud_foreign_key
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0515_slugify_invalid_monitors"),
+    ]
+
+    state_operations = [
+        migrations.RemoveField(
+            model_name="pagerdutyservice",
+            name="organization_integration_id",
+        ),
+        migrations.AddField(
+            model_name="pagerdutyservice",
+            name="organization_integration",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                default=-1,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="sentry.organizationintegration",
+                db_constraint=False,
+            ),
+            preserve_default=False,
+        ),
```

Comment:
We don't have to backfill this field before removing the old one?

Reply:
This "add field" isn't a real column, it's merely the *relation* to the Organization Integration.  Under the hood, it uses the existing column `organization_integration_id`.